src/Files.App/Data/Factories/PropertiesNavigationViewItemFactory.cs

```diff
@@ -107,7 +115,10 @@ public static ObservableCollection<NavigationViewItemButtonStyleItem> Initialize
 					PropertiesNavigationViewItems.Remove(securityItem);
 
 				if (!hashItemEnabled)
+				{
 					PropertiesNavigationViewItems.Remove(hashesItem);
+					PropertiesNavigationViewItems.Remove(signaturesItem);
```

Comment:
Is there ever a time when the hashes page is displayed, but signature should be hidden?

Reply:
Good call 😄
I added a check to show the section only for supported file types

BTW, when an item has no signature, should we show a message like below or leave the section empty?
<img width="792" height="548" alt="image" src="https://github.com/user-attachments/assets/b777c4df-a04a-443b-ac06-1176c176d273" />

Reply:
Not showing anything might look like a bug, I think it's best to show a message.

Reply:
FYI https://github.com/files-community/Files/pull/17334/commits/571d0a2f12ae9a3fabfb3f5ec4065776794ff33b already shows the message

Reply:
@mdtauk do you have any suggestions for the empty state?